### PR TITLE
Update: MoheshwarAmarnathBiswas.UsefulToolsForWindows version 1.2

### PR DESCRIPTION
**Pull request by fluentmoheshwar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80924)

### CODE REVIEW

File: manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.UsefulToolsForWindows
+PackageVersion: 1.1
```
Comment (Trenly) [inline manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml:5]:
```suggestion
PackageVersion: 1.2
```
Version in control panel is 1.2

File: manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.UsefulToolsForWindows
+PackageVersion: 1.1
```
Comment (Trenly) [inline manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.2
```
Version in control panel is 1.2

File: manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.UsefulToolsForWindows
+PackageVersion: 1.1
```
Comment (Trenly) [inline manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.yaml:5]:
```suggestion
PackageVersion: 1.2
```
Version in control panel is 1.2

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion should match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>

File: manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml (hunk 1 above)
Comment (fluentmoheshwar) [reply manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml:5]: Oh no I messed up the installer itself. it should be 1.1. Anyway What should I do now? Should I rebuild the installer or just edit the package version?

Comment (fluentmoheshwar) [reply manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml:5]: I am doing rename cuz I am too lazy to rebuild installer.

**Review by Trenly — APPROVED** [review]

**Pull request merged**